src/controller.js

```diff
@@ -17,6 +17,7 @@ import { insistCapData } from './capdata';
 import { parseVatSlot } from './parseVatSlots';
 
 const evaluateOptions = makeDefaultEvaluateOptions();
+evaluateOptions.shims.unshift('this.globalThis = this');
```

Comment:
This feels a little weird, but I think you said `globalThis` was somewhat standard (or at least fairly common?).

Is there any way this might reveal a global to code that couldn't previously reach it? At least that global will be frozen, right?

Reply:
This is standard enough, but probably should be made part of `lockdown` when we move to that model.  For now, it doesn't expose anything since we're just creating a (to-be-frozen) reference to the same global object that all user code has access to.

I've added a comment indicating this.